buildstockbatch/postprocessing.py

```diff
@@ -18,6 +18,7 @@
 from dask.dataframe.io.parquet import create_metadata_file
 from fsspec.implementations.local import LocalFileSystem
 from functools import partial
+from gcsfs import GCSFileSystem
```

Comment:
postprocessing.py is used by all our implementations. `gcsfs` is only installed if the `gcp` extras are installed. This will cause import errors for all other implementations where they may not have installed those extras. It looks like none of these imported fsspec filesystem classes are even used in this file. We can just remove the imports.

```suggestion
```

Reply:
Good call - fixed!